### PR TITLE
Adding DAML

### PR DESCRIPTION
**Pull request by aria-ml**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

@conda-forge/help-python

### CODE REVIEW

**Comment by aria-ml** [comment]
@conda-forge/help-python ready for review :)

File: azure-pipelines.yml [hunk 1]
```diff
@@ -12,4 +12,4 @@ pr:
 jobs:
   - template: ./.azure-pipelines/azure-pipelines-linux.yml
   - template: ./.azure-pipelines/azure-pipelines-osx.yml
-  - template: ./.azure-pipelines/azure-pipelines-win.yml
```
Comment (xhochy) [inline azure-pipelines.yml:15]: Please don't touch this file.

File: recipes/daml/conda_build_config.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+python:
```
Comment (xhochy) [inline recipes/daml/conda_build_config.yaml:1]: You should not be specifying the Python via this technique. We have a global pinning for that.

File: recipes/daml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - scikit-learn
+    - pytorch
+    - torchvision
+    - torchmetrics
+    - tensorflow
+    - tensorflow-probability
+
+test:
+  imports:
+    - daml
```
Comment (xhochy) [inline recipes/daml/meta.yaml:39]:
```suggestion
    - daml
  requires:
    - pip
  commands:
    - pip check
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by aria-ml** [comment]
@conda-forge/help-python @xhochy ready for review, thanks for taking a look!

File: recipes/daml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+
+requirements:
+  build:
```
Comment (xhochy) [inline recipes/daml/meta.yaml:19]:
```suggestion
  host:
```

File: recipes/daml/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    - python
```
Comment (xhochy) [inline recipes/daml/meta.yaml:20]: This needs a minimal Python version pin

**Review by xhochy — CHANGES_REQUESTED** [review]

**Review by xhochy — APPROVED** [review]

**Pull request merged**